packages/core/inject-version.ts

```diff
@@ -1,13 +1,14 @@
-import { writeFileSync } from "fs";
+import { writeFileSync, readFileSync } from "fs";
 import { dirname, resolve } from "path";
 import { fileURLToPath } from "url";
-import packageJSON from "./package.json" assert { type: "json" };
 
 const __filename = fileURLToPath(import.meta.url);
 const __dirname = dirname(__filename);
 
 const VERSION_FILE_PATH = "src/version.ts";
 
+const packageJSON = JSON.parse(readFileSync("./package.json").toString());
```

Comment:
Stops tsc from having json import issues.

Reply:
oh weird... it shouldn't be having issues since we have `"resolveJsonModule" : true` in compiler options and the `assert `on the import. what version of TS are you on?